metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -17,50 +17,34 @@
 
 package org.apache.kafka.controller;
 
-import java.util.Arrays;
```

Comment:
Do we really need to move these imports from above to below?

Reply:
I can revert it. I use the auto-import tool in Intellij which likes a specific order.